huaweicloud/services/organizations/resource_huaweicloud_organizations_policy.go

```diff
@@ -0,0 +1,259 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product Organizations
+// ---------------------------------------------------------------
+
+package organizations
+
+import (
+	"context"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourcePolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourcePolicyCreate,
+		UpdateContext: resourcePolicyUpdate,
+		ReadContext:   resourcePolicyRead,
+		DeleteContext: resourcePolicyDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the name to be assigned to the policy.`,
+			},
+			"content": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the policy text content to be added to the new policy.`,
+			},
+			"type": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the type of the policy to be created.`,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
```

Comment:
remove `Computed:    true`  if the description can be updated to empty

Reply:
done